sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -61,14 +62,17 @@ export default {
       type: Number,
       required: true,
     },
-    locationName: {
-      type: String,
-      required: true,
-    },
   },
-  setup() {
+  setup(props: any) {
+    const store = useStore();
     const tab = ref("coe-tab");
-    return { EnrollmentPeriod, COETab, tab };
+
+    const locationName = computed(() => {
```

Comment:
This logic can be moved to a composable. Please refer to useStudentStore.ts as example.

Reply:
Sure, I will update.